setup.py

```diff
@@ -48,22 +48,30 @@
         'Topic :: Software Development :: Libraries :: Python Modules',
     ],
     install_requires = [
-        'Django==1.8.13',
-        'django-autocomplete-light==2.3.3',
-        'django-extensions>=1.5.5',
-        'django-nose>=1.4',
+        'Django>=1.8.2',
+        'django-extensions>=1.5.5',  # Assuming admin.py is in powerdns/
         'dj.choices>=0.10.0',
-        'mysqlclient==1.3.7',
-        'nose-cov>=1.6',
-        'factory_boy>=2.5.2',
         # 3.3.3 includes bug, https://github.com/rtfd/readthedocs.org/issues/2101
-        'djangorestframework==3.3.2',
-        'django-rest-swagger==0.3.8',
-        'django-filter>=0.10.0',
         'django-threadlocals>=0.8',
-        'docutils>=0.12',
         'rules>=0.4',
-        'raven==5.20.0'
+    ],
+    extras_require = {
+        'docs': [
+            'docutils>=0.12',
+        ],
+        'dnsaas': [
```

Comment:
could you describe in some documentation (i don't know how up-to-date it is, but lets try) new installation commands?